src/tests/ftest/container/attribute.py

```diff
@@ -94,18 +123,25 @@ def test_container_large_attributes(self):
         Test description: Test large randomly created container attribute.
 
         :avocado: tags=container,container_attr,attribute,large_conattribute
```

Comment:
Probably you should remove the tag 'container_attr' since you are defining a new common one (container_attribute) below.

Reply:
Is there a way to search if a tag is being used by some process/automation? I'd like to avoid deleting a tag that is being used.

Reply:
Hmmm, probably locally with `./launch.py --list container_attr`

Reply:
Alright, let me give it a look

Reply:
Running avocado list --paginator=off --filter-by-tags=container_attr ./
Detected tests:
./container/attribute.py

Seems it is safe to implement your suggestion (y)